packages/features/eventtypes/components/EventTypeDescription.tsx

```diff
@@ -40,6 +42,31 @@ export const EventTypeDescription = ({
 
   const paymentAppData = getPaymentAppData(eventType);
 
+  const showManagedBadge = eventType.metadata?.managedEventConfig && !isPublic;
+  const numOfManagedByAdmins = eventType.managedBy?.admins.length || 0;
```

Comment:
Add a better variable name, no need to include managed. Change it to `numberOfTeamAdmins`

Reply:
done